tracee-rules/main.go

```diff
@@ -27,6 +28,9 @@ func (realClock) Now() time.Time {
 }
 
 func main() {
+	// TODO Introduce --pprof flag?
+	defer profile.Start(profile.CPUProfile, profile.ProfilePath("."), profile.NoShutdownHook).Stop()
```

Comment:
I think we need a flag --pprof in tracee-rules to conditionally enable profiling. Exposing /debug/pprof is also very useful to look into other things such as memory, coroutines or mutex

Reply:
I'd say lets do it in a different PR

Reply:
Agree. Reverted that